### ISSUE TITLE
Aplastic zygomatic arch

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7524]
**Preferred term label:**
Aplastic zygomatic arch

**Synonyms**

**Definition (free text, please give PubMed ID)**
Developmental defect characterized by the absence of the zygomatic arch.

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
OMIM:200110
PMID: 26119818
2/10

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
